src/main/java/uk/gov/hmcts/reform/idam/client/models/AuthenticateUserResponse.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.idam.client.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+
+@Getter
```

Comment:
can we just use public final fields?

Reply:
Since this is a library I would go for adding a getter, its least surprise and won't confuse people.
Some info here:
https://stackoverflow.com/a/6927823/4951015

Reply:
ok, fair point.